source/extensions/stat_sinks/dog_statsd/config.h

```diff
@@ -9,6 +9,9 @@ namespace Extensions {
 namespace StatSinks {
 namespace DogStatsd {
 
+// DogStatsD compatible statsd sink
+extern constexpr char DogStatsd[] = "envoy.stat_sinks.dog_statsd";
```

Comment:
this all looks much better, thank you!

sorry, one more nit: can you rename this constant to be "DogStatsdName"? Same with the other constants.

Reply:
Sure, I will rename all.